.github/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
Good job adding linters to your project. 👍  But currently your linters are not doing their work because the folder system needs to be `.github/workflows/linters.yml`. Kindly add the workflows folder. 👍